base/files/file_unittest.cc

```diff
@@ -644,17 +650,29 @@ TEST(FileTest, MAYBE_WriteDataToLargeOffset) {
 
   const char kData[] = "this file is sparse.";
   const int kDataLen = sizeof(kData) - 1;
+  int64_t kLargeFileOffset;
+
 #if defined(STARBOARD)
-#if SB_IS(32_BIT)
```

Comment:
This condition doesn't work for Windows, and I have to replace with condition if (sizeof(long) == 4).
Otherwise tests on Windows would fail because of data overflow.